### ISSUE TITLE
Unneccessary code

### ISSUE THREAD

**Issue by vineetbansal** [ComputationalCryoEM/ASPIRE-Python#34]
This method is duplicating everything that `evaluate` for a `CTFFilter` does:

https://github.com/ComputationalCryoEM/ASPIRE-Python/blob/8eb7f2c1bc8645fd3f649d4ac3c22417cb1c3827/src/aspire/utils/filters.py#L221

It's semantically equivalent to calling `evaluate` with a 1D `omega` vector instead of a 2D `omega` matrix and will return exactly the following (I've verified this):

```
self.evaluate(np.vstack([kvals * 2 * np.pi, np.zeros_like(kvals)]))
```

We should simply find out why/where the `2*np.pi` factor comes in, and (if needed), have `evaluate` of radial filters work on 1D inputs as well as 2D.

**Comment by vineetbansal** [comment]
Support for evaluating 1d omega vectors (for radial filters) added in ca9ef2e655b8787bdb576de94e5843cecfeee358.

`evaluate_k` now simply calls `evaluate` while raising a `DeprecationWarning`. This issue should be closed once `evaluate_k` is deleted.

**Comment by junchaoxia** [comment]
This issue have been solved in PR #62.
